wit-definitions/aggregator/wit/aggregator.wit

```diff
@@ -0,0 +1,100 @@
+package wavs:aggregator@0.6.0-alpha.1;
+
+use wavs:types/core@0.6.0-alpha.1 as core-types;
+use wavs:types/service@0.6.0-alpha.1 as service-types;
+use wavs:types/chain@0.6.0-alpha.1 as chain-types;
+
+interface helpers {
```

Comment:
I think we can get rid of this helpers and keep things DRY:

1. move `log-level` into `core.wit`
2. move `service-and-workflow-id` and `workflow-and-workflow-id` into `service.wit`